zkevm-circuits/src/mpt_circuit/witness_row.rs

```diff
@@ -154,6 +158,7 @@ pub(crate) enum MptWitnessRowType {
     StorageNonExisting = 19,
 }
 
+/// MPT witness row
```

Comment:
I think `MptWitnessRowType` and `MptWitnessRow` can also completely be removed no?